db/internal_stats.cc

```diff
@@ -1056,6 +1063,63 @@ void InternalStats::DumpDBStats(std::string* value) {
   db_stats_snapshot_.write_stall_micros = write_stall_micros;
 }
 
+void InternalStats::DumpDBMapStats(
+    std::map<std::string, std::string>* db_stats) {
+  double seconds_up = (env_->NowMicros() - started_at_ + 1) / kMicrosInSec;
+  double interval_seconds_up = seconds_up - db_stats_snapshot_.seconds_up;
+  (*db_stats)["seconds_up"] = ToString(seconds_up);
```

Comment:
Are these strings like "seconds_up" just created here, or is it shared with some other places?